### ISSUE TITLE
Couldn't load jp2 (JPEG2000) image.

### ISSUE THREAD

**Issue by Kitof** [ZaaLabs/ZaaIL#1]
I'm unable to load any jp2 (jpeg 2000) image. I've even tried with TestImages.

**Comment by natebeck** [comment]
I don't believe that JP2 support was compiled in with the build we released... but you can feel free to add support for it in yourself.  The process we followed to create ZaaIL is documented here: http://www.zaalabs.com/2010/04/compiling-with-alchemy-1/
